lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/TestHunspellRepositoryTestCases.java

```diff
@@ -36,9 +37,18 @@
  */
 @RunWith(Parameterized.class)
 public class TestHunspellRepositoryTestCases {
+  private static final Set<String> IGNORED =
+      Set.of(
+          "hu", // Hungarian is hard: a lot of its rules are hardcoded in Hunspell code, not aff/dic
```

Comment:
Hungarian is hard, isn't it! :)

Reply:
The irony is that "hun" in "hunspell" means it was initially created for Hungarian, but its declarative options still aren't enough to describe it :(

Reply:
Whenever I come across Finno-Ugric languages I always end up with a conclusion that they must be alien descendants left over on Earth... (and I have friends speaking Hungarian!).

Reply:
Yes, they're quite unusual. I've studied Finnish a bit, and it was fun.